pipelines/broad/arrays/imputation_beagle/ImputationBeaglePreChunk.wdl

```diff
@@ -5,26 +5,17 @@ import "../../../../tasks/broad/Utilities.wdl" as utils
 
 workflow ImputationBeagle {
 
-    String pipeline_version = "0.0.1"
```

Comment:
do we want to use pipeline_version somehow? like put this into the header of the output files?

Reply:
oh im a dumb dumb, this is actually necessary for a warp wdl to be valid. im gonna add it back (we'll have to up the version when we merge to develop).  Even still I think versioning should be tracked by the github release tag + the tsps pipeine that was run, not necessarily this value.  Something we shold talk about though